api-report/test-runtime-utils.api.md

```diff
@@ -154,7 +162,7 @@ export class MockContainerRuntimeFactoryForReconnection extends MockContainerRun
 
 // @public
 export class MockContainerRuntimeForReconnection extends MockContainerRuntime {
-    constructor(dataStoreRuntime: MockFluidDataStoreRuntime, factory: MockContainerRuntimeFactoryForReconnection, overrides?: {
+    constructor(dataStoreRuntime: MockFluidDataStoreRuntime, factory: MockContainerRuntimeFactoryForReconnection, runtimeOptions?: MockContainerRuntimeOptions, overrides?: {
```

Comment:
if we're trying to avoid breaking changes, `runtimeOptions` should probably be after` overrides` (even though typical usage doesn't use this constructor directly)

Reply:
It's not breaking, but the current usage pattern forced me to add `{}` for overrides (as it's not currently as used) in two call sites so I opted to put it before the overrides. No strong feelings one way or the other, this seemed to minimize changes.